third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -35,6 +35,10 @@ use std::{
 pub struct LiveVarAnnotation(BTreeMap<CodeOffset, LiveVarInfoAtCodeOffset>);
 
 impl LiveVarAnnotation {
+    pub fn new(annot: BTreeMap<CodeOffset, LiveVarInfoAtCodeOffset>) -> Self {
```

Comment:
doc please.

Reply:
Also, are there any preconditions expected of the input parameter? If so, it would be nice to document it here.

Reply:
Done. But I wonder if the doc is actually useful because the implementation is just `Self(annot)`... The only precondition should be that there are no holes in the map? @wrwg